ebms-send-in/src/main/kotlin/no/nav/emottak/fellesformat/FellesFormatWrapper.kt

```diff
@@ -32,9 +38,9 @@ private fun createFellesFormatMottakEnhetBlokk(sendInRequest: SendInRequest): EI
         herIdentifikator = sendInRequest.addressing.from.partyId.getIdentifikatorByType("HER")
         orgNummer = sendInRequest.addressing.from.partyId.getIdentifikatorByType("orgnummer", "ENH")
         meldingsType = "xml"
-        partnerReferanse = sendInRequest.cpaId
+        this.partnerReferanse = partnerReferanse
```

Comment:
Ingen stor greie, men kunne satt denne direkte her. Noe lignende:
```
partnerReferanse = when (sendInRequest.addressing.service) {
        "PasientlisteForesporsel" -> sendInRequest.partnerId?.toString() ?: ""
        else -> sendInRequest.cpaId
    }
```

Reply:
Det var vurdert, men hvis vi gjør det for dette her, og så det samme for flere andre felt i ettertid, blir det fort rotat i koden.